adapters/automatad/automatadtest/exemplary/simple-banner.json

```diff
@@ -0,0 +1,111 @@
+{
+	"mockBidRequest": {
+	  "id": "test-request-id",
+	  "imp": [
+		{
+		  "id": "test-banner-imp-id",
+		  "banner": {
+			"format": [
+			  {
+				"w": 300,
+				"h": 250
+			  }
+			]
+		  },
+		  "ext": {
+			"bidder": {
+			  "position": "123",
+			  "placementId":"a34gh6d"
+			}
+		  }
+		}
+	  ],
```

Comment:
The formatting on these test JSON files seem to be a little off. Can you please fix that?

Reply:
I recommend using the Beautify plugin in VS Code, but any formatter will do.

Reply:
Formatted all json files.